admin/database/postgres/postgres.go

```diff
@@ -734,6 +734,23 @@ func (c *connection) CheckUserIsAProjectMember(ctx context.Context, userID, proj
 	return res, nil
 }
 
+func (c *connection) GetCurrentTrialOrgCount(ctx context.Context, userID string) (int, error) {
+	var count int
+	err := c.getDB(ctx).QueryRowxContext(ctx, "SELECT current_trial_orgs_count FROM users WHERE id=$1", userID).Scan(&count)
+	if err != nil {
+		return 0, parseErr("org count", err)
+	}
+	return count, nil
+}
+
+func (c *connection) IncrementCurrentTrialOrgCount(ctx context.Context, userID string) error {
+	_, err := c.getDB(ctx).ExecContext(ctx, "UPDATE users SET current_trial_orgs_count = current_trial_orgs_count + 1 WHERE id=$1", userID)
+	if err != nil {
+		return parseErr("org count", err)
+	}
+	return nil
+}
```

Comment:
Could this be computed with a join query instead of tracked separately? It seems it won't be called often, so even if the query is inefficient it should be fine.

If not, shouldn't it be decremented when a trial expires or is upgraded?

Reply:
The reason its being tracked separately because this quota is across the lifetime of user and once they have started two trial orgs then they cannot have another one even if previous ones were upgraded to paid plan. So no way to know at the moment how many orgs were on trial in past, user can directly upgrade to paid plan without going through trial and in that case this won't be incremented. Same reason why we don't need to decrement this.

Reply:
Okay, didn't realize it's a lifetime thing.

Was it a request to make it a lifetime thing? Since we will probably forget to change it for each internal user, it sounds like this will lead to quota issues when folks are doing live demoes and POCs in the future.

Reply:
Yes it was a requirement from product cc @ericokuma